### ISSUE TITLE
Microphone sound cast, audio not?

### ISSUE THREAD

**Issue by 123Maul** [RogueAmoeba/Soundflower#54]
I'm using Soundflower combined with Soundcast to cast audio to my Chromecast Audio. (MBP OS X El Capitan 10.11.4)
Since a while (not sure how long) it stopped working. I've tried re-installing Soundflower and Soundcast, but no luck: When I play audio and select Soundflower 2ch as output, no sound comes through my speakers connected to the CC Audio. 
But I just noticed a strange thing: When starting the Soundcast app, both input and output are set to Soundflower audio device. I tried setting 'input' back to my Built-in microphone, and suddenly the sounds that my microphone records are immediately played through my speakers. 
All the while I'm playing music that should be cast, but this doesn't arrive at my speakers.
Anyone has an idea what might cause this?
Using Chrome browser and casting audio (e.g. youtube Tab to the CC Audio) works flawlessly..
Thanks in advance :)

**Comment by mattingalls** [comment]
yes, you can't have input and output set to SF - this creates a feedback loop.  i don't know soundcast, but  the general use case is setting system default output device to SF, then whatever app you are using has input set to SF (and either muted output or a different output device!)  if you need to monitor the audio out of the default device use a multi-output aggregate device -- i have some steps how to do that here: https://github.com/mattingalls/Soundflower/releases
